recipe/meta.yaml

```diff
@@ -10,7 +10,7 @@ source:
     sha256: dceb15fef556a2775121bcc695561df4ff0e09248e0ce3a2d58f5244a9f61421
 
 build:
-    number: 0
+    number: 1
```

Comment:
I think it is necessary since we are updating the recipe without updating the version.  It is safest to update the build number rather than overwriting existing packages on anaconda.org